.bazelci/presubmit.yml

```diff
@@ -19,6 +20,7 @@ basic_example_bzlmod: &basic_example_bzlmod
     - "--incompatible_enable_android_toolchain_resolution"
     - "--android_platforms=//:arm64-v8a,//:x86"
     - "--enable_bzlmod=True"
```

Comment:
I think we could just get rid of --enable_bzlmod and --enable_workspace=false here, since bzlmod is the default for all versions of rules_android_ndk tested by BazelCI.